pom.xml

```diff
@@ -31,4 +31,27 @@
     <build>
     </build>
 
+    <profiles>
+        <profile>
+            <id>release</id>
```

Comment:
this profile was added to set the -Xdoclint:none parameter for the maven-javadoc-plugin. This parameter is needed because of the new DocLint feature in java 8 which caused some errors regarding javadoc comments. It wasn't added directly in the community pom because java 7 does not have this option and it will throw errors when releasing any project that uses java 7.